os/pm/pm_idle.c

```diff
@@ -26,8 +26,12 @@
 #include <tinyara/clock.h>
 #include <tinyara/irq.h>
 #include <tinyara/arch.h>
+#include "../kernel/sched/sched.h"
```

Comment:
Please use `<tinyara/sched.h>`

Reply:
no, the definition needed is defined in os/kernel/sched/sched.h. If using the path you mentioned, it will face compilation error.